### PR TITLE
Electron: Allow screenshare to work with camera video

### LINKED ISSUE #976: Include video and screen sharing in the screen sharing doc
**Business case**: Enable developers to send  screen and camera streams to a channel. 

**Subject Matter Expert**: 
  - *Technical*: 
  - *Language*: Iain
  - *PM*: only if essential

**Scope**: (https://docs.agora.io/en/video-calling/develop/product-workflow), use the example in https://docs-preprod.agora.io/en/live-streaming-premium-4.x/screensharing_web_ng?platform=Web#enable-both-screen-sharing-and-video to add a step in the screen sharing part to show how to send camera and screen video streams to a channel in parallel.
  - [ ] Android: @saudsami 
  - [ ] iOS: @nirm2009 
  - [ ] Web: @hussain-khalid 
  - [ ] macOS: @nirm2009 
  - [ ] Windows: @hussain-khalid 
  - [x] Electron: @Kishan-Dhakan 
  - [ ] Unity: @nirm2009 
  - [ ] Flutter: @saudsami 
  - [x] React-Native: @Kishan-Dhakan 

**Acceptance criteria**: Developers must be able to easily send multiple video streams to a channel.

**Additional valuable information**:

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
- Resolves #976

### CODE REVIEW

File: shared/video-sdk/develop/product-workflow/project-implementation/electron.mdx [hunk 1]
```diff
@@ -67,33 +74,16 @@ To implement volume control and screen sharing in your <Vpl k="CLIENT" />, take
...
-        channelOptions.publishScreenTrack = publishScreenTrack
-        agoraEngine.updateChannelMediaOptions(channelOptions);
-    }
-    ```
 
-4.  **Display the screen sharing stream locally**
+3.  **Display the screen sharing stream locally**
 
-    Setup a `VideoCanvas` and use it in the `setupLocalVideo` method to show the screen track locally. To switch between the screen stream and the camera stream, in `preload.js`, add the following method before `window.onload = () => `:
+    Setup a `VideoCanvas` and use it in the `setupScreenSharing` method to show the screen track locally. The new `VideoCanvas` enables you to publish your screen track along with your local video. In `preload.js`, add the following method before `window.onload = () => `:
```
Comment (atovpeko) [inline shared/video-sdk/develop/product-workflow/project-implementation/electron.mdx:80]:
```suggestion
    Set up a `VideoCanvas` and use it in the `setupScreenSharing` method to show the screen track locally. The new `VideoCanvas` enables you to publish your screen track along with your local video. In `preload.js`, add the following method before `window.onload = () => `:
```

File: shared/video-sdk/develop/product-workflow/project-implementation/electron.mdx [hunk 2]
```diff
@@ -67,33 +74,16 @@ To implement volume control and screen sharing in your <Vpl k="CLIENT" />, take
...
-        let channelOptions = new ChannelMediaOptions();
-        channelOptions.publishMicrophoneTrack = publishScreenTrack;
-        channelOptions.publishCameraTrack = !publishScreenTrack;
-        channelOptions.publishScreenTrack = publishScreenTrack
-        agoraEngine.updateChannelMediaOptions(channelOptions);
-    }
-    ```
 
-4.  **Display the screen sharing stream locally**
+3.  **Display the screen sharing stream locally**
```
Comment (atovpeko) [inline shared/video-sdk/develop/product-workflow/project-implementation/electron.mdx:78]: Would it be possible to use either stream or track everywhere?

**Review by atovpeko — CHANGES_REQUESTED** [review]
A few small things

File: shared/video-sdk/develop/product-workflow/project-implementation/electron.mdx (hunk 2 above)
Comment (Kishan-Dhakan) [reply shared/video-sdk/develop/product-workflow/project-implementation/electron.mdx]: That's a good point, resolved

**Review by atovpeko — APPROVED** [review]

**Pull request merged**